easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-8.0-CrayGNU-21.09.eb

```diff
@@ -0,0 +1,41 @@
+# mkra (CSCS), 2020
+easyblock = 'Binary'
+
+name = 'OpenFOAM'
+version = '8.0'
+
+homepage = 'http://www.openfoam.org/'
+description = """
+    OpenFOAM is a free, open source CFD software package. OpenFOAM has an 
+    extensive range of features to solve anything from complex fluid flows
+    involving chemical reactions, turbulence and heat transfer, to solid 
+    dynamics and electromagnetics.
+"""
+
+toolchain = {'name': 'CrayGNU', 'version': '21.09'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/%(name)s/%(name)s-%(version_major)s/archive']
+sources = [{'download_filename': '20210316.tar.gz', 'filename': SOURCE_TAR_GZ}]
+patches = ['patch.%(name)s-%(version)s-20210316']
+
+builddependencies = [
+    ('CMake/3.14.5', EXTERNAL_MODULE),
+]
+dependencies = [
+    ('cray-tpsl', EXTERNAL_MODULE),
+]
+
+buildininstalldir = 'True'
+modtclfooter = """
+    setenv FOAM_BASH %(installdir)s/%(name)s-8-20210316/etc/bashrc
+"""
+extract_sources = 'True'
+install_cmd = "source etc/bashrc && ./Allwmake -j 6 "
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['%(name)s-8-version-8/platforms/linux64GccDPInt32Opt/bin', '%(name)s-8-version-8/platforms/linux64GccDPInt32Opt/lib'],
```

Comment:
```suggestion
    'dirs': ['%(name)s-8-20210316/platforms/linux64GccDPInt32Opt/bin', '%(name)s-8-version-8/platforms/linux64GccDPInt32Opt/lib'],
```

Reply:
The sanity check path needs to be adapted to the current release: one might also define a locale variable `local_release = 20210316` and use it throughout the script instead of hard-coding the local release.